tests/run-tests

```diff
@@ -130,24 +119,38 @@ _up() {
 }
 
 _playbook() {
-    case $mode in
-        host)
-            inventory=inventory.cfg
-            ;;
-        aux)
-            inventory=../example/inventory.cfg
-            ;;
-        *)
-            echo _playbook called in guest mode
-            return 1
-            ;;
-    esac
-    ansible-playbook \
-        -i $inventory \
-        -e @vars-iri.yaml \
-        -e @../example/secrets.yaml \
-        ../example/playbook.yaml \
-        "$@"
+  build=""
+  extraflags=""
+  for arg in "$*"; do
+    if [[ $arg == "build" ]]; then
+      build="--build"
+      extraflags="-e run_update_script=yes"
+    fi
+  done
+
+  case $mode in
+    host)
+      inventory=inventory.cfg
+      ansible-playbook \
+          -i $inventory \
+          -e @vars-iri.yaml \
+          -e @../example/secrets.yaml \
+          ../example/playbook.yaml ${extraflags}
+      ;;
+    aux)
+      mkdir ${builddir}/dlconfig
+      cp ../example/* ${builddir}/dlconfig
+      cp ../example/secrets.yaml ${builddir}
```

Comment:
This is kind of crufty, but I can't think of a better way at the moment. I think I want to stop using secrets.yaml, in favor of a certificate generated on the server and installed in bitbucket via the HTTP API. With secrets.yaml eliminated, we could probably do without this copy. But that's for another PR.

Reply:
Hmm. Leaving secrets.yaml in the project directory and adding it to .gitignore is a tempting solution. But I generally assume that a git repo with no uncommitted changes can be wiped out and re-cloned without disrupting anything; it would be annoying if someone did that and then found they no longer had a copy of their secrets file. (Annoying but not fatal--you would just generate and install new keys.)

Reply:
I think moving secrets.yaml would be more trouble than it's worth, as many partners already have it set up this way.